### ISSUE TITLE
Adjust tabular format (design styles and fonts)_Projects Component Development

### ISSUE THREAD

**Issue by satyaaneesh98** [TeamCodeCrafterss/CodeCrafters#44]
The initial code for the tabular data visualization is a sample dev case, further the data representation format like design styles, font and look have to be further modified as per the design and develop it for impressive UI.

**Comment by satyaaneesh98** [comment]
I have tried few design styles and fonts and table adjustments. I will be pushing the code for this once I have done with the add, edit, delete button additions initially which are part of my sprint-3 planning. Because adding these buttons is creating collisions for my design when I tested for one. This task code will not be final, because the add, edit, delete and search functionalities will launch a sidebar and UI changes which will also affect my look. So, I will have one more task created after this initial code submission so that will have the final code containing the fix for all the functionalities as well.
